service/create/pki_test.go

```diff
@@ -0,0 +1,85 @@
+package create
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestGetClusterCA(t *testing.T) {
+	svc, _ := New(DefaultConfig())
+
+	tests := []struct {
+		name           string
+		certCommonName string
+		expectedCA     string
```

Comment:
This is the common name of the CA, right? Would be cool if the variable name reflects that somehow.